### PR TITLE
aruha-778 Correct scopes for cursor arithmetics endpoints

### PR DESCRIPTION
**Pull request by lmontrieux**
The following endpoints need the nakadi.event_stream.read scope:
* POST /event-type/{name}/shifted-cursors
* POST /event-type/{name}/cursor-distance
* POST /event-type/{name}/cursors-lag

### CODE REVIEW

File: src/main/java/org/zalando/nakadi/config/SecurityConfiguration.java [hunk 1]
```diff
@@ -60,6 +60,9 @@ public void configure(final HttpSecurity http) throws Exception {
                     .antMatchers(GET, "/subscriptions/*/cursors/**").access(hasScope(eventStreamReadScope))
                     .antMatchers(PUT, "/subscriptions/*/cursors/**").access(hasScope(eventStreamReadScope))
                     .antMatchers(GET, "/subscriptions/*/stats/**").access(hasScope(eventStreamReadScope))
+                    .antMatchers(POST, "/event-types/*/shifted-cursors").access(hasScope(eventStreamReadScope))
+                    .antMatchers(POST, "/event-types/*/cursors-lag").access(hasScope(eventStreamReadScope))
+                    .antMatchers(POST, "/event-types/*/cursor-distances").access(hasScope(eventStreamReadScope))
```
Comment (v-stepanov) [inline src/main/java/org/zalando/nakadi/config/SecurityConfiguration.java]: can you please add these endpoints to test `AuthenticationTest`?

Comment (zrobgar) [inline src/main/java/org/zalando/nakadi/config/SecurityConfiguration.java]: 👍

**Comment by zrobgar** [comment]
👍

File: src/main/java/org/zalando/nakadi/config/SecurityConfiguration.java (hunk 1 above)
Comment (lmontrieux) [inline src/main/java/org/zalando/nakadi/config/SecurityConfiguration.java]: @v-stepanov ce39c84

**Comment by v-stepanov** [comment]
👍

**Comment by lmontrieux** [comment]
👍

**Pull request merged**